perfkitbenchmarker/linux_packages/hadoop.py

```diff
@@ -200,6 +205,8 @@ def _RenderConfig(vm,
   # This determines the number of reduce tasks in Terasort and is critical to
   # scale with the cluster.
   num_reduce_tasks = reduces_per_node * num_workers
+  if _BLOCKSIZE.value():
```

Comment:
This is should just be `.value:` (no parentheses).

Reply:
Resolved